app.py

```diff
@@ -897,8 +937,16 @@ def tree_process(tree : ParentedTree, senttok: List[str]) -> tuple[ParentedTree,
 			# if initial parse labels non-punctuation as punctuation, change to N-Head
 		if (not is_possible_punct_token(senttok[i])) and (is_punct_label(subt.label)):
 			subt.label = 'N-Head'
-		ptree_terminals.append(subt)
+	return tree_copy
 
+def ptree_to_cgel(tree: ParentedTree, senttok: List[str]) -> ParentedTree:
```

Comment:
Can this be folded in as a method of `ActiveDopTree`? Or is there reason to keep it separate?